packages/type/README.md

```diff
@@ -85,7 +85,7 @@ what we call type styles. These tokens have a variety of properties for styling
 how text is rendered on a page.
 
 You can find a full reference of the type styles that are available
-[here](https://next.carbondesignsystem.com/guidelines/typography/productive) .
+[Carbon Design System website](https://www.carbondesignsystem.com/guidelines/typography/productive) .
```

Comment:
```suggestion
[here](https://www.carbondesignsystem.com/guidelines/typography/productive) .
```

Was the text meant to be changed or, if so, should we update the sentence?

Reply:
Oh I mean to update the sentence completely, to avoid having link text that just says "here"

Reply:
Oh sorry just noticed your edit --

Here's two quick resources I found to explain this: 
- https://www.w3.org/QA/Tips/noClickHere
- https://www.w3.org/TR/WCAG10-HTML-TECHS/#link-text

In addition to the reasons about context provided in those links, if you are using a screenreader + tabbing between focusable elements you may just hear "Link Here" or "Link Click here" which isn't helpful.